pages/getting-started/understand-odata-in-6-steps.html

```diff
@@ -1421,4 +1433,60 @@ <h4>Contribute to "Understanding OData in 6 steps"</h4>
 
         });
     }
+
+    // Select all text in a modal when Ctrl+A is pressed
+    $(document).keydown(function(e) {
+        // If the 'Ctrl + a' or 'Ctrl + Shift + a' for Narrator key are pressed simultaneously.
+        // If the condition is met, it prevents the default action and selects the contents of any visible modal's body.
+        // Check if the modal is open
+        if ($('.modal:visible .modal-body').length && e.ctrlKey && (e.key === 'a' || e.key === 'A')) {
+            e.preventDefault();
+            var modalBody = $('.modal:visible .modal-body');
+            modalBody.focus();
+
+            // Create a range to select the content within the modal body
+            var range = document.createRange();
+            range.selectNodeContents(modalBody[0]);
+
+            // Get the current selection and remove any existing ranges
+            var selection = window.getSelection();
+            selection.removeAllRanges();
+
+            // Add the new range to the selection
+            selection.addRange(range);
+        }
+    });
+
+    // Copy code to clipboard
+    function copyCode() {
+        // Loop through each modal
+        document.querySelectorAll('.modal').forEach(function(modal) {
+            // Check if the modal is currently open
+            if (modal.style.display === 'block') {
+                var body = modal.querySelector('.modal-body');
+
+                var range = document.createRange();
+                range.selectNodeContents(body);
+                // Get the current selection and remove any existing ranges
+                var selection = window.getSelection();
+                selection.removeAllRanges();
+                // Add the new range to the selection
+                selection.addRange(range);
+
+                document.execCommand("copy");
+
+                // Provide feedback to the user
+
+                $('#feedback').text('Response code copied to clipboard.').show();
```

Comment:
```suggestion
                $('#feedback').text('Copied to clipboard.').show();
```

"Response code" has an entirely different meaning. We could say "Payload copied to clipboard" or "Response copied to clipboard" - If "Copied to clipboard" doesn't feel sufficient

Reply:
`Payload copied to clipboard` sounds good.